### DISCUSSION TITLE
I need help with ad compatibility depending on the phone

### DISCUSSION THREAD

**Discussion by AdamT-hub** [Poing-Studios/Godot-AdMob-Android-iOS#218] [Q&A]
(I am using admob plugin on godot 3.5.2) You see, my problem is that I did all the steps correctly and when I run it on my Samsung Galaxy A32 the test ads work perfectly but when I run it on my Xiaomi 13c the ads do not work and the buttons in the test scene are all grayed out (in the output of the test scene there is also no output). Do I need to ask for internet permission from the user? or is the problem something else?

**Comment by gumaciel** [comment]
Hi, this seems to be a problem out of the scope of this plugin. 

There is already an issue opened for this: #219
